### PR TITLE
remove external_system_identifier logic from template and add to wrappers.py

### PR DESCRIPTION
**Pull request by Dinika**
## Description
The pr also removes styling from the `View in` label so that it's colour is black (like every other label in detailed view)
screenshot from 2017-11-24 15-24-13
(Image 2 below shows how the label looks currently. It has a navy blue colour)

Currently, the `View in` label appears even when there's no external_system_id, as shown below:
screenshot from 2017-11-24 15-17-32

This PR removes this label
screenshot from 2017-11-24 15-24-48

## Motivation and Context
Uniform styling for all labels 

## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: inspirehep/modules/theme/jinja2filters.py [hunk 1]
```diff
@@ -590,6 +590,12 @@ def is_external_link(url):
             ('.pdf', '.png', '.jpg', '.jpeg')))
 
 
+@blueprint.app_template_filter()
+def is_institute(institute):
+    """Checks if given string is an institute."""
+    return institute.lower() in ['kekscan', 'ads', 'cds', 'hal', 'msnet',
+                                 'osti', 'zblatt', 'euclid']
```
Comment (jacquerie) [inline inspirehep/modules/theme/jinja2filters.py]: Wrong rebase: this died in https://github.com/inspirehep/inspire-next/commit/e35c380ba093ca6a78e71b5443d6c6cbc21bbf46, it should stay dead.

File: inspirehep/modules/theme/jinja2filters.py [hunk 2]
```diff
@@ -590,6 +590,13 @@ def is_external_link(url):
             ('.pdf', '.png', '.jpg', '.jpeg')))
 
 
+@blueprint.app_template_filter()
+def is_institute(institute):
```
Comment (michamos) [inline inspirehep/modules/theme/jinja2filters.py]: the name is wrong, we don't call this an institute any more in the schema. Maybe `is_known_exernal_system_identifiers`? but the way this whole thing is done is wrong IMHO.
Why don't you pull the whole logic to link to external system identifiers in the record wrapper, and have there a property which is a `dict` `link text: link url` and only exposes linkable `external_system_identifiers`?
Right now, if you want to add a new one (e.g. `HepData` is missing), you need to amend both this function and the template.

Comment (Dinika) [reply inspirehep/modules/theme/jinja2filters.py]: @jmartinm @michamos I've made some changes to this PR as per your suggestions. Most of the logic is now in wrappers.py

**Comment by jacquerie** [comment]
Was this superseded by https://github.com/inspirehep/inspire-next/pull/3026, @Dinika ?

**Comment by Dinika** [comment]
> Was this superseded by #3026, @Dinika ?
It was. I forgot to close this one. Will do it now.

**Pull request closed without merging**